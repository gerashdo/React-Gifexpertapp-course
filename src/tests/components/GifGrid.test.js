import { shallow } from 'enzyme';
import { GifGrid } from '../../components/GifGrid';

describe( 'Pruebas del componente GifGrid', () => {

    test('Debe renderizar correctamente el componente <GifGrid />', () => {
        
        const categoria = 'Death Note';

        const wrapper = shallow(
            <GifGrid categoria={categoria} />
        );

        expect( wrapper ).toMatchSnapshot();

    });
    

});