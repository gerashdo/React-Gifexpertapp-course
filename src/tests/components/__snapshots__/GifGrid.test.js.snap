// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente GifGrid Debe renderizar correctamente el componente <GifGrid /> 1`] = `
<Fragment>
  <h3
    className="animate__animated animate__fadeIn"
  />
  <p
    className="animate__animated animate__flash"
  >
    Loading...
  </p>
  <div
    className="card-grid"
  />
</Fragment>
`;
